src/app/Dashboard/styles/dashboard.scss

```diff
@@ -0,0 +1,146 @@
+.wrapper {
+  display: flex;
+  justify-content: space-around;
+}
+
+.main-content {
+  display: flex;
+  position: absolute;
+  top: 0;
+  z-index: 1;
+  width: 90%;
+}
+
+.sub-content {
+  flex-grow: 100;
+  font-family: 'DIN Pro';
+  padding-top: rem(23px);
+}
+
+.user-dashboard {
+  margin-top: rem(27px);
+}
+.user-dashboard__name {
+  @include text-styles(rem(27px), $lightAccent, 500, rem(0.56px), rem(27px));
+}
+
+.user-dashboard__level {
+  height: rem(23px);
+  width: rem(23px);
+  padding: rem(1px);
+  margin: rem(15px) 0;
+  @include text-styles( rem(14px), $orange, 500, rem(0.48px), rem(18px));
+  border: rem(1px) solid $orange;
+}
+
+.profile-overview {
+  display: flex;
+  justify-content: space-between;
+}
+
+
+.profile-overview__image {
+  height: 134px;
+  width: 122px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(../../../images/person-placeholder.jpg);
+}
+
+.user-dashboard__actions {
+  display: flex;
+  margin-top: 45px;
+  padding-left: 14px;
+  justify-content: space-between;
+
+  .user-dashboard__title {
+    @include text-styles(rem(16px), $lightAccent, normal, rem(0.33px), rem(21px));
+  }
+
+  span {
+    text-align: center;
+    @include text-styles(rem(14px), $lightAccent, 300, normal, rem(24px));
+  }
+
+  .fa {
+    color: $lightAccent !important;
+  }
+
+  .fa-plus {
+    padding-right: 22.5px; 
+  }
+
+  .fa-filter {
+    padding-left: 8px;
+  }
+
+  .button__add {
+    height: 32px;
+    width: 143px;
+    background-color: #F5A623;
+    box-shadow: inset 0 1px 3px 0 rgba(0,0,0,0.5), 0 12px 35px 0 rgba(52,77,178,0.34);
+  }
+
+  .button__filter {
+    height: 32px;
+    width: 78px;
+    margin-left: 26px;
+    border: 1px solid #FFFFFF;
+    background-color: transparent;
+  }
+}
+
+// Large devices (desktops/mac, 992px and up)
+@media (min-width: 992px) and (max-width: 1199.98px) { }
```

Comment:
Great job with these media queries.  I have been wondering whether it would be worth it moving these to their own file to ensure that we have these styles separated and avoid this file getting too large. Again, your thoughts appreciated.